customizations/README.md

```diff
@@ -379,6 +382,38 @@ externalRedis:
     #   -----END CERTIFICATE-----
 ```
 
+### Enable BigQuery Oauth connections
+
+This feature allows users to create a BigQuery connection using `Sign in with Google` instead of providing a service account key. Note that connections created with Oauth cannot be shared with other organization users.
+
+#### Previous steps
+
+1. Create an oauth consent screen inside the desired GCP project.
+   - Introduce an app name and a user support email.
+   - Add an authorized domain (the one used in your email).
+   - Add another email as dev contact info (it can be the same).
+   - Add the following scopes: `./auth/userinfo.email`, `./auth/userinfo.profile` & `./auth/bigquery`.
+
+2. Create an Oauth credentials.
+   - Type: Web application.
+   - Authorized JavaScript origins: `https://<your_selfhosted_domain>`.
+   - Authorized redirect URIs: `https://<your_selfhosted_domain>/connections/bigquery/oauth`.
+   - Download the credentials file.
+
+#### Custom configuration
+
+Add the following lines to your `customizations.yaml` and populate them with the credential's file corresponding values:
```

Comment:
> and populate them with the credential's file corresponding values

This is a bit confusing, please refer to the proper documentation step, like https://github.com/CartoDB/carto-selfhosted-helm/blob/main/customizations/README.md#how-to-apply-the-configurations